pkg/aws/eni/node.go

```diff
@@ -174,7 +174,7 @@ func (n *Node) ReleaseIPs(ctx context.Context, r *ipam.ReleaseAction) error {
 func (n *Node) PrepareIPAllocation(scopedLog *logrus.Entry) (a *ipam.AllocationAction, err error) {
 	limits, limitsAvailable := n.getLimits()
 	if !limitsAvailable {
-		return nil, fmt.Errorf("Unable to determine limits")
+		return nil, fmt.Errorf(errUnableToDetermineLimits)
```

Comment:
I think moving to sentinel error values is fine, but no need to `fmt.Errorf()` again

Reply:
sounds good, what should I use then as `errUnableToDetermineLimits` is a string? 🤔

Reply:
Yeah you can do:

```
var errUnableToDetermineLimits = errors.New("...")
```

Reply:
I reverted those changes as they are not related to this change anyway!